Tasks/FileTransformV2/package.json

```diff
@@ -18,9 +18,12 @@
   "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
   "dependencies": {
     "@types/mocha": "2.2.48",
```

Comment:
Could you please update mocha types also to v5.2.7?

Reply:
done